### ISSUE TITLE
Request new rule: Declare properties before nested rules

### ISSUE THREAD

**Issue by spawnia** [kristerkari/stylelint-scss#165]
Properties in a rule should always be declared before nested rules. This is because the properties for the element itself are most important to see first.

This would be considered a violation:

```css
.bar{
  color: red;
  .foo {
    color: pink;
  }
}
```
This would be considered OK
```css
.bar{
  .foo {
    color: pink;
  }
  color: red;
}
```

**Comment by kristerkari** [comment]
Thanks @spawnia! At first I was thinking that this could be a good rule to add, but then I remembered that you can actually already do this kind of linting with `stylelint-order`. Have a look at the examples:
https://github.com/hudochenkov/stylelint-order/blob/master/rules/order/README.md
